minst/model.py

```diff
@@ -1,15 +1,24 @@
-
+import copy
 import json
 import jsonschema
+import logging
 import pandas as pd
 import os
+from sklearn.cross_validation import train_test_split
 
 import minst.utils as utils
 
+logger = logging.getLogger(__name__)
+
+
+class MissingDataException(Exception):
+    pass
+
 
 class Observation(object):
     """Document model each item in the collection."""
```

Comment:
Kay I'll update it ... for now, an observation in this instance is an observed 'new note event'. granted the duration is implicit. sooo we mark onsets as the start of observing a note event, and yes, an observation is an implied abstraction from this event.

sry, I don't understand your final comment?